src/main/java/by/exadel/internship/repository/FormRepository.java

```diff
@@ -47,7 +47,8 @@
 
     @EntityGraph(attributePaths = {"interview"})
     @Query("select distinct f from Form f " +
-            " where f.deleted = false and f.interview.admin.id = :userId or f.interview.techSpecialist.id = :userId")
+            " where f.deleted = false and f.interview.admin.id = :userId " +
+            "or f.deleted = false and f.interview.techSpecialist.id = :userId")
```

Comment:
may be better - `where f.deleted = false and f.interview.admin.id = :userId " +            "or f.deleted = false and f.interview.techSpecialist.id = :userId`

Reply:
this was exactly the case, but then the deletion belongs to the first part "or"